creator-node/src/components/healthCheck/healthCheckController.js

```diff
@@ -1,14 +1,16 @@
 const express = require('express')
+const { promisify } = require('util')
+
+const config = require('../../config')
 const { handleResponse, successResponse, errorResponseBadRequest, handleResponseWithHeartbeat } = require('../../apiHelpers')
-const { healthCheck, healthCheckDuration } = require('./healthCheckComponentService')
-const { syncHealthCheck } = require('./syncHealthCheckComponentService')
+const { trackFileUpload } = require('../../fileManager')
```

Comment:
any reason to use this vs handleTrackContentUpload?

Reply:
I tried using `handleTrackContentUpload`, but this wasn't able to pass the error back up to the original call from `healthCheckFileUploadController` because the response gets sent before I can add headers to it.

To use `handleTrackContentUpload` we might require changes to it or `trackFileUpload`. I think this might have unintended consequences elsewhere.